src/ai/backend/manager/registry.py

```diff
@@ -2691,6 +2696,8 @@ async def _update() -> None:
                                 "version": agent_info["version"],
                                 "compute_plugins": agent_info["compute_plugins"],
                                 "architecture": agent_info.get("architecture", "x86_64"),
+                                "abuse_report_path": abuse_report_path,
+                                "auto_terminate": auto_terminate,
```

Comment:
Is there a specific reason to store these config values to database instead of retriving on-demand with RPC?

Reply:
Storing these values in db seems better than retriving them with RPC cause the values do not change much. They only changes when the agent's status changes. But using RPC needs to retreive them with RPC every request.
If this idea is bad, I will change this code. How do you think?